lib/FusionInventory/Agent/Task/Inventory/Virtualization/Lxc.pm

```diff
@@ -190,6 +196,7 @@ sub  _getVirtualMachines {
 
         my $uuid = getVirtualUUID($machineid, $hostname);
         $container->{UUID} = $uuid if $uuid;
+	$container->{NAME} = $container->{UTS_NAME} if $container->{UTS_NAME};
```

Comment:
I would suggest you to always remove UTS_NAME as it is not expected in VIRTUALMACHINE structure and set NAME only if UTS_NAME matches numbers.
```suggestion
        my $utsname = delete $container->{UTS_NAME};
        $container->{NAME} = $utsname if $utsname && $container->{NAME} =~ /^\d+$/;
```

Reply:
Yes you're right, it's better like that.
Tests passed without problems.